client/src/client.rs

```diff
@@ -632,31 +657,7 @@ pub mod transaction {
     }
 }
 
-/// URI that `Client` uses to route outgoing requests.
-//TODO: remove duplication with `iroha_core::torii::uri`.
-pub mod uri {
-    //! Module with uri constants
-
-    /// Query URI is used to handle incoming Query requests.
-    pub const QUERY: &str = "/query";
-    /// Instructions URI is used to handle incoming ISI requests.
-    pub const TRANSACTION: &str = "/transaction";
-    /// URI for configuration checking
-    pub const CONFIGURATION: &str = "/configuration";
-    /// Block URI is used to handle incoming Block requests.
-    pub const CONSENSUS: &str = "/consensus";
-    /// Health URI is used to handle incoming Healthcheck requests.
-    pub const HEALTH: &str = "/health";
-    /// Metrics URI is used to export metrics according to [Prometheus
-    /// Guidance](https://prometheus.io/docs/instrumenting/writing_exporters/).
-    pub const METRICS: &str = "/metrics";
-    /// The URI used for block synchronization.
-    pub const BLOCK_SYNC: &str = "/block";
-    /// The web socket uri used to subscribe to pipeline and data events.
-    pub const SUBSCRIPTION: &str = "/events";
-    /// Get pending transactions.
-    pub const PENDING_TRANSACTIONS: &str = "/pending_transactions";
-}
+use iroha_core::torii::uri;
```

Comment:
shouldn't this be at the top of the file? 
Why were api endpoints duplicated in both torii and client?

Reply:
> shouldn't this be at the top of the file?

No problem. 

> Why were api endpoints duplicated in both torii and client?

The only thing that I needed to change was add a `/` to some format statements. So probably some miscommunication.